### ISSUE TITLE
How to create album with command line

### ISSUE THREAD

**Issue by ManuCart** [3846masa/upload-gphotos#182]

**Comment by 3846masa** [comment]
CLI can't only create album.
But, when uploading photo with `-a` flag, create album if not exists.

**Comment by ManuCart** [comment]
OK, thanks for your help

**Comment by ManuCart** [comment]
Well now I have this error when apps try to creat album name

```
Username: 
Password:
Success to login!
UserID is 
atParam is 

Uploading [==========================================================================================================] 100% 0.0s

Uploaded successfully!
Album "Nom de l'album" is not found.
TypeError: Cannot use 'in' operator to search for '76647426' in 2
    at Function.parseInfo (/usr/local/lib/node_modules/upload-gphotos/lib/Photo.js:237:63)
    at /usr/local/lib/node_modules/upload-gphotos/lib/index.js:452:52
    at Array.map (native)
    at GPhotos.<anonymous> (/usr/local/lib/node_modules/upload-gphotos/lib/index.js:451:36)
    at next (native)
    at onFulfilled (/usr/local/lib/node_modules/upload-gphotos/node_modules/co/index.js:65:19)
```

**Comment by 3846masa** [comment]
Oh, I'll fix. Thanks for your report.

**Comment by 3846masa** [comment]
I released v1.3.3.
Please wait for building via CI.

**Comment by ManuCart** [comment]
Great! working as I expected
